docs/source/get_started/computers.rst

```diff
@@ -203,79 +200,101 @@ The configuration of computers happens in two steps.
         aprun -n {tot_num_mpiprocs}
         poe
 
-   * **Text to prepend to each command execution**: This is a multiline string,
-     whose content will be prepended inside the submission script before the
-     real execution of the job. It is your responsibility to write proper ``bash`` code!
-     This is intended for computer-dependent code, like for instance loading a
-     module that should always be loaded on that specific computer. *Remember*
-     *to end the input by pressing* ``<CTRL>+D``.
-     A practical example::
+   * **Default number of CPUs per machine**: The number of MPI processes per machine that
+     should be executed if it is not otherwise specified. Use ``0`` to specify no default value. 
+   
+   At the end, the command will open your default editor on a file containing a summary
+   of the configuration up to this point, and the possibility to add ``bash``
+   commands that will be executed either *before* the actual execution of the job
+   (under 'pre-execution script') or *after* the script submission (under 'Post execution script').
+   These additional lines need may set up the environment on the computer,
+   for example loading modules or exporting environment variables, for example::
 
         export NEWVAR=1
         source some/file
 
-     A not-to-do example::
+   .. note:: Don't specify settings here that are specific to a code, calculation or scheduler -- 
+      you can set further pre-execution commands at the ``Code`` and ``CalcJob`` level.
 
-       #PBS -l nodes=4:ppn=12
+   When you are done editing, save and quit (e.g. ``<ESC>:wq<ENTER>`` in ``vim``).
+   The computer has now been created in the database but you still need to *configure* access to it
+   using your credentials.
 
-     (it's the plugin that will do this!)
+   For the user convenience, it is also possible provide some (or all) the information
```

Comment:
```suggestion
   In order to avoid having to retype the setup information the next time round, it is also possible provide some (or all) of the information
```